testsuite/forge/src/backend/local/cargo.rs

```diff
@@ -160,7 +160,12 @@ pub fn git_merge_base<R: AsRef<str>>(rev: R) -> Result<String> {
 pub fn cargo_build_common_args() -> Vec<&'static str> {
     let use_release = use_release();
     let consensus_only = build_consensus_only_node();
-    let mut args = vec!["build", "--features=failpoints,indexer"];
+    let without_indexer = std::env::var("FORGE_BUILD_WITHOUT_INDEXER").is_ok();
```

Comment:
nit: can you wrap `std::env::var("FORGE_BUILD_WITHOUT_INDEXER").is_ok()` into a function so it's consistent with `use_release` and `consensus_only`?

Reply:
rearranged, and added comments about how weird it is that compile options for forge's inner `aptos-node` are locked in at compile time for `forge`, except the new option I'm adding which sets compile options for aptos-node when we're building aptos-node.